deploy/crds/crd-certificates.yaml

```diff
@@ -229,6 +229,18 @@ spec:
                       type: array
                       items:
                         type: string
+                    extraNames:
+                      description: ExtraNames is a list of pkix.ExtraNames to be set on the Certificate.
+                      type: array
+                      items:
+                        type: object
+                        properties:
+                          type:
+                            description: The type. This should be an asn1.ObjectIdentifier string, e.g., "2.5.4.42"
+                            type: string
+                          value:
+                            description: The value. This should be an IA5String
```

Comment:
What's an IA5String? Can you provide some examples, references etc in this documentation?

Reply:
Basically, it's ASCII (which is the US implementation of IA5), but with a couple of small differences. Here's the best article I could find to explain it. https://www.zytrax.com/tech/ia5.html. I can put a comment in here that says something to that effect.